.vscode/settings.json

```diff
@@ -2,7 +2,7 @@
     "editor.formatOnSave": true,
     "[python]": {
         "editor.codeActionsOnSave": {
-            "source.organizeImports": true
```

Comment:
VSCode does this automatically for version 1.85+. I could toggle it back, but we're just going to keep fighting this same change re-appearing and most devs should be on 1.85+ (current version is 1.93).